src/index.js

```diff
@@ -2,28 +2,28 @@ import 'assets/styles/main.scss';
 import VideoPlayer from './video-player';
 import { assign } from 'utils/assign';
 
-
-function videoPlayer(id, playerOptions, ready = null) {
-  return new VideoPlayer(id, playerOptions, ready);
+if (window.cloudinary && window.cloudinary.Cloudinary) {
+  console.error('For version 2.0.0 and above, cloudinary-core is not needed for using the Cloudinary Video Player');
```

Comment:
I think a `console.warning` is enough here

Reply:
indeed